packages/experiments/src/components/SelectedItemsList/ControlledSelectedItemsList.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+
+import { IControlledSelectedItemsList, IControlledSelectedItemListProps, BaseSelectedItem } from './SelectedItemsList.types';
+import { copyToClipboard } from '../../utilities/copyToClipboard';
+import { useSelectionIndices } from '../../utilities/useSelectionIndices';
+
+/**
+ * An externally-managed selected item list.
+ */
+export const ControlledSelectedItemsList = <TItem extends BaseSelectedItem>(props: IControlledSelectedItemListProps<TItem>) => {
+  const selection = React.useMemo(() => props.selection, [props.selection]);
```

Comment:
This memo shouldn't be needed anymore, since this just references props.selection. In a prior revision, ControlledSelectedItemsList could manage selection itself.